### ISSUE TITLE
Browserify optimisation fails

### ISSUE THREAD

**Issue by pseudonom** [purescript-contrib/pulp#23]
`pulp browserify` works fine. 

`pulp browserify --optimise` results in `psc: Main: openFile: does not exist (No such file or directory)`

`pulp browserify --optimise -m "src/Main.purs"` results in 
`Module 'Main' has been defined more than once` after trying to exec `psc` with an argument list like

``` javascript
[ '--module',
  'src/Main.purs',
  '--main',
  'src/Main.purs',
  ...
  'src/Main.purs',
  ...
```

**Comment by pseudonom** [comment]
I noticed that the necessary fix for this is already in `main`. But it's not in `2.1.0` which is the latest version available on npm at the moment. Would it be possible to get a new version on npm? Thanks.

**Comment by bodil** [comment]
Just published 2.2.0, so I assume this is resolved.

**Comment by pseudonom** [comment]
Thanks.
